Symfony/CS/Fixer/Contrib/RandomApiMigrationFixer.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * This fixer in general identical to AliasFunctionsFixer but aims
+ * to modernize random api calls, therefore it's separated fixer.
+ *
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class RandomApiMigrationFixer extends AbstractFixer
+{
+    /** @var string[] stores older (key) - newer (value) functions mapping */
+    private static $replacements = array(
+        'rand' => 'mt_rand',
+        'srand' => 'mt_srand',
+        'getrandmax' => 'mt_getrandmax',
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isTokenKindFound(T_STRING);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        foreach ($tokens->findGivenKind(T_STRING) as $index => $token) {
+            // skip expressions without parameters list
+            $nextToken = $tokens[$tokens->getNextMeaningfulToken($index)];
+            if (!$nextToken->equals('(')) {
+                continue;
+            }
+
+            // skip expressions which are not function reference
+            $prevTokenIndex = $tokens->getPrevMeaningfulToken($index);
+            $prevToken = $tokens[$prevTokenIndex];
+            if ($prevToken->isGivenKind(array(T_DOUBLE_COLON, T_NEW, T_OBJECT_OPERATOR, T_FUNCTION))) {
+                continue;
+            }
+
+            // handle function reference with namespaces
+            if ($prevToken->isGivenKind(array(T_NS_SEPARATOR))) {
+                $twicePrevTokenIndex = $tokens->getPrevMeaningfulToken($prevTokenIndex);
+                $twicePrevToken = $tokens[$twicePrevTokenIndex];
+                if ($twicePrevToken->isGivenKind(array(T_NEW, T_STRING, CT_NAMESPACE_OPERATOR))) {
+                    continue;
+                }
+            }
```

Comment:
For this, I'd think of fair PSI-tree instead of tokens, but this too big piece of work for all of us here.

For short-term I could just create an Utility-class - when all PRs are merged. But it will be just this two blocks on lines 52-67.